.travis.base.yml

```diff
@@ -0,0 +1,38 @@
+os: linux
```

Comment:
travis no longer offer supports for opensource project.
need to migrate this to Github Actions, or something else...